plugins/cue_points/quiz/lib/api/filters/base/KalturaAnswerCuePointBaseFilter.php

```diff
@@ -10,6 +10,8 @@
 	(
 		"parentIdEqual" => "_eq_parent_id",
 		"parentIdIn" => "_in_parent_id",
+		"userEntryIdEqual" => "_eq_user_entry_id",
```

Comment:
change property names to quizUserEntryId to be consistent with the model property names